### ISSUE TITLE
Peer Review

### ISSUE THREAD

**Issue by annasxu** [fjt37/ml-big-data-project#6]
This project is about creating a model to predict the price of BitCoin by using historical prices, volumes, hashrates, and more. 

I think the topic is relevant and interesting, and overall the group did a good job with the midterm report. I like how the group mentioned the feedback from the project proposal and L1 regularization from class. I think the graphs with Mean Error on the x-axis were very insightful and well chosen. 

A couple questions and concerns I had were: When you looked at that graph of when the model was just starting to memorize the data, was that on the training data, the test data, or the whole dataset? When I read through your proposal, I see no mention of a training set, so I'm wondering how you came to those conclusions. Also, on the graph of using the optimal days, you guys came to the conclusion of using 14 days, but when I look at the graph, it looks like 2 days has an even lower error. How do you justify using 14 instead of 2? Finally, I really like that you guys are looking into legislation, because I think it will hugely impact the BitCoin prices, but I still wonder logistically how you can incorporate those more environmental/political features. Perhaps it would be good to look at other BitCoin price prediction papers, to get ideas!
